superset/connectors/sqla/views.py

```diff
@@ -243,17 +244,13 @@ class TableModelView(DatasourceModelView, DeleteMixin, YamlExportMixin):  # noqa
         'is_sqllab_view': _('SQL Lab View'),
         'template_params': _('Template parameters'),
     }
+    validators_columns = {
```

Comment:
What about SQL Lab tables?

Reply:
@fabianmenges could you be more specific? It potentially seems wrong to use a period in a datasource name.